test/asserts/plain-object-ts-assert.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { Assert: BaseAssert, Violation } = require('validator.js');
+const ObjectAssert = require('../../src/asserts/plain-object-ts-assert');
+
+/**
+ * Extend `Assert` with `ObjectAssert`.
+ */
+
+const Assert = BaseAssert.extend({
+  Object: ObjectAssert
+});
+
+/**
+ * Test `ObjectAssert`.
+ */
+
+describe('PlainObjectTSAssert', () => {
+  it('should throw an error if the input value is not an object', () => {
+    const choices = ['foobar', 123];
+
+    choices.forEach(choice => {
+      try {
+        Assert.plainObjectTS().validate(choice);
+
+        fail();
+      } catch (e) {
+        expect(e).toBeInstanceOf(Violation);
+      }
+    });
+  });
+
+  it('should expose `assert` equal to `plainObjectTS`', () => {
+    try {
+      Assert.plainObjectTS().validate('FOO');
+
+      fail();
+    } catch (e) {
+      expect(e.show().assert).toBe('PlainObjectTS');
+    }
+  });
+
+  it('should accept a object', () => {
+    Assert.plainObjectTS().validate({ foo: 'bar' });
```

Comment:
Here we should add a test case for the TS scenario, otherwise this will be the same test suite as the `PlainObjectAssert`.